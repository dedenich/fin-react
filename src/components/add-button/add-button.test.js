import React from 'react';
import renderer from 'react-test-renderer';

import AddButton from './add-button.jsx';


it(`AddButton renders correctly`, () => {
  const tree = renderer
  .create(
      <AddButton/>
  )
  .toJSON();
  expect(tree).toMatchSnapshot();
});
