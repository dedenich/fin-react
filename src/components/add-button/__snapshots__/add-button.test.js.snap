// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddButton renders correctly 1`] = `
<button
  className="add_button"
  onClick={[Function]}
  type="button"
/>
`;
